include/deal.II/lac/linear_operator.h

```diff
@@ -1118,6 +1371,228 @@ linear_operator(const OperatorExemplar &operator_exemplar, const Matrix &matrix)
 
 //@}
 
+namespace SchurComplement
```

Comment:
Once the `Payload` mechanism has been fully developed, I foresee this functionality being embedded therein. I have put this here for now so that the functionality provided here does not appear to clash with that already provided by the `SchurMatrix` class.